setup.py

```diff
@@ -32,6 +32,7 @@
     install_requires=[
         'redis',
         'celery',
+        'packaging',
```

Comment:
I can't recall is packaging included by default in newer python releases?  I'm -0 on adding a dependency for such a minimal "feature".  Would it be possible to sniff the version of Celery by doing something like

"""
  CELERY_4_OR_GREATER = True
   try:
       import SomethingThatOnlyExistsInCelery4
    except ImportError:
       CELERY_4_OR_GREATER = False
"""

I'm not sure that is any better but would like to hear other's thoughts.

Reply:
Coming in from outside, I've encountered this issue. You don't need to add packaging as a dependency at all.

You should probably use `from pkg_resources import parse_version` as pkg_resources is part of setuptools and will (almost) always be provided in the field.

Reply:
Either way, this definitely an issue. Would be great to accept this PR vs. no fix at all.

Reply:
`celery.VERSION` is a tuple of ints that can be be used for comparisons without needing to install `packaging`. `packaging` is not installed by default, but might be there anyway because of newer pip releases or other packages using it.

Reply:
@fangpenlin the @lambacck  solution seems to be the simplest one:
 
```diff
diff --git a/redbeat/schedulers.py b/redbeat/schedulers.py
index a2e0ffe..06f0746 100644
--- a/redbeat/schedulers.py
+++ b/redbeat/schedulers.py
@@ -15,7 +15,7 @@ try:
 except ImportError:
     import json

-from celery import __version__ as celery_version
+from celery import VERSION as CELERY_VERSION
 from celery.beat import Scheduler, ScheduleEntry, DEFAULT_MAX_INTERVAL
 from celery.utils.log import get_logger
 from celery.signals import beat_init
@@ -36,7 +36,7 @@ from .decoder import (
     from_timestamp, to_timestamp
     )

-CELERY_4_OR_GREATER = StrictVersion(celery_version) >= StrictVersion('4.0')
+CELERY_4_OR_GREATER = CELERY_VERSION[0] >= 4


 def ensure_conf(app):
```